packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1208,7 +1208,6 @@ async function handler(
   const organizerOrganizationProfile = await prisma.profile.findFirst({
     where: {
       userId: organizerUser.id,
-      username: dynamicUserList[0],
```

Comment:
Important change. For team events, _organizerOrganizationProfile_ would always be null because dynamicUserList[0] would be teamSlug.

This never caused any problem because, for team events we were using different way to determine bookingUrl.

Now I need to use organizerOrganizationProfile to detemine the username correctly for team as well as user events.


<img width="1624" height="556" alt="Image" src="https://github.com/user-attachments/assets/32b64dc3-888b-405e-a17f-29969e98bf47" />